app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,13 @@
+class NotificationsController < ApplicationController
+
+  def index
+    # @notifications = Notification.unread_for_user current_user.id
+  end
+
+  def show
+    notification = Notification.find params[:id]
+    notification.is_read = true
+    notification.save
```

Comment:
`#save!`

I also think you can put this logic into its own method. Say `Notification#mark_as_read`.